### PR TITLE
docs: Add issue template for new validation rules

### PR DESCRIPTION
**Pull request by barbeau**
Adds a new issue template option to the "New issue" workflow.  The options after this is merged are:
1. Bug report
2. Feature request
3. New validation rule

Closes https://github.com/MobilityData/gtfs-validator/issues/816

### CODE REVIEW

**Comment by barbeau** [comment]
Sorry, please hold on review. The GitHub UI duplicated the file - I'll remove it.

**Comment by barbeau** [comment]
@lionel-nj Ok, should be good for review now.

File: .github/ISSUE_TEMPLATE/new-validation-rule.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+name: New validation rule
+about: Suggest a new GTFS validation rule for this project
+title: ''
+labels: new rule
+assignees: ''
+
+---
+
+**What problem in GTFS datasets does this new rule address? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (lionel-nj) [inline .github/ISSUE_TEMPLATE/new-validation-rule.md:11]:
```suggestion
<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --> 
```

File: .github/ISSUE_TEMPLATE/new-validation-rule.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+**Describe the new validation rule**
+A clear description of the new rule logic you'd like to see. Ex. For all stops in stops.txt, [...]
```
Comment (lionel-nj) [inline .github/ISSUE_TEMPLATE/new-validation-rule.md:14]:
```suggestion
<!-- A clear description of the new rule logic you'd like to see. Ex. For all stops in stops.txt, [...] -->
```

File: .github/ISSUE_TEMPLATE/new-validation-rule.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**Error vs warning**
+Should this be an error (violates a requirement of the GTFS spec) or a warning (a best practice or suggestion)?
```
Comment (lionel-nj) [inline .github/ISSUE_TEMPLATE/new-validation-rule.md:17]:
```suggestion
<!-- Should this be an error (violates a requirement of the GTFS spec) or a warning (a best practice or suggestion)? -->
```

**Review by lionel-nj — CHANGES_REQUESTED** [review]
Thanks @barbeau! I suggest using `<-- "text" --> markers as suggested by @Cristhian-HA in https://github.com/MobilityData/metrics/pull/6#discussion_r592746746: 

> adding those symbols will prevent for displaying in the issue itself and will reduce the noise

We can open a new PR to update the existing templates to adopt this.

File: .github/ISSUE_TEMPLATE/new-validation-rule.md (hunk 3 above)
Comment (lionel-nj) [inline .github/ISSUE_TEMPLATE/new-validation-rule.md:17]: 👍🏾

File: .github/ISSUE_TEMPLATE/new-validation-rule.md (hunk 1 above)
Comment (barbeau) [reply .github/ISSUE_TEMPLATE/new-validation-rule.md:11]: @lionel-nj I actually think this makes it much harder to read when you're creating the issue, especially if it's your first time creating an issue:

image

I'd be inclined to err on the side of making it easier for people to read when they are writing the issue - I think we'll get better information. People should really just be deleting this text anyway and replacing it with their own as it's just a placeholder - that's what I normally do, and what I've seen happen on most GitHub repositories that I work with.

What do you think?

Comment (lionel-nj) [reply .github/ISSUE_TEMPLATE/new-validation-rule.md:11]: Alright! Let's stick to what is most commonly done then!

**Review by lionel-nj — APPROVED** [review]
LGTM

**Pull request merged**